src/foam/u2/view/PasswordView.js

```diff
@@ -36,10 +36,23 @@ foam.CLASS({
     }
   `,
 
+  constants: [
+    {
+      type: 'String',
+      name: 'VISIBILITY',
+      value: '/foam2/src/foam/u2/images/visibility.svg'
+    },
+    {
+      type: 'String',
+      name: 'VISIBILITY_OFF',
+      value: '/foam2/src/foam/u2/images/visibility-off.svg'
+    }
+  ],
+
   properties: [
     {
       name: 'visibilityIcon',
-      value: '/foam2/src/foam/u2/images/visibility.svg'
+      value: '/foam2/src/foam/u2/images/visibility-off.svg'
```

Comment:
Isn't this redundant, since you have the same value as a constant?
Would this be better:
  factory: function() { return this.VISIBILITY_OFF; }

Reply:
OK